internal/ethapi/api.coreth_test.go

```diff
@@ -0,0 +1,130 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ethapi
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+type testSuggestPriceOptionsBackend struct {
+	Backend // embeds the interface to avoid implementing unused methods
```

Comment:
supernit (aka feel free to discard this conversation) otherwise it sounds like `Backend` embeds, but it's just "we" or `testSuggestPriceOptionsBackend`

```suggestion
	Backend // embed the interface to avoid implementing unused methods
```